zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/LocalSessionTracker.java

```diff
@@ -45,4 +49,7 @@ public long createSession(int sessionTimeout) {
         return sessionId;
     }
 
+    public Set<Long> localSessions() {
+        return (sessionsWithTimeouts == null) ? Collections.emptySet() : sessionsWithTimeouts.keySet();
```

Comment:
Unless I am missing something, and just FYI, `sessionsWithTimeouts` cannot be `null` here as it would have caused the `super` constructor to throw.

Reply:
good catch, I removed the check.